runtime/README.md

```diff
@@ -601,6 +601,59 @@ Table: `SHUTDOWN` output arguments
 | chksum       | u32       | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips_status  | u32       | Indicates if the command is FIPS approved or an error
 
+### ADD\_SUBJECT\_ALT\_NAME
+
+Provides a subject alternative name otherName. Whenever CERTIFY_KEY_EXTENDED is called with the 
+DMTF_OTHER_NAME flag after ADD_SUBJECT_ALT_NAME is called, the resulting DPE CSR or leaf certificate 
+will contain a subject alternative name extension containing the provided otherName, which must be a 
+DMTF device info. All such certificates produced by CERTIFY_KEY_EXTENDED will continue to have the 
+DMTF otherName subject alternative name extension until reset.
+
+Command Code: `0x414C_544E` ("ALTN")
+
+*Table: `ADD_SUBJECT_ALT_NAME` input arguments*
+
+| **Name**     | **Type** | **Description**
+| --------     | -------- | ---------------
+| chksum       | u32      | Checksum over other input arguments, computed by the caller. Little endian.
+| dmtf\_device\_info\_size  | u32   | The size of the DMTF Device Info UTF8String.
+| dmtf\_device\_info | u8[128]   | The DMTF Device Info UTF8String.
+
+*Table: `ADD_SUBJECT_ALT_NAME` output arguments*
+
+| **Name**    | **Type** | **Description**
+| --------    | -------- | ---------------
+| chksum      | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
+| fips\_status | u32      | Indicates if the command is FIPS approved or an error.
+
+### CERTIFY\_KEY\_EXTENDED
+
+Produces a DPE leaf certificate or CSR containing custom extensions provided by the SoC.
+
+Command Code: `0x434B_4558` ("CKEX")
+
+*Table: `CERTIFY_KEY_EXTENDED` input arguments*
```

Comment:
nit: can you align the columns so it's easier to read in raw markdown?

Reply:
Done. I went ahead and aligned all the other columns in the readme too.